CMakeLists.txt

```diff
@@ -123,6 +133,70 @@ if(USE_GPU)
     ADD_DEFINITIONS(-DUSE_GPU)
 endif(USE_GPU)
 
+if(USE_CUDA)
+    find_package(CUDA REQUIRED)
```

Comment:
> Deprecated since version 3.10: Superseded by first-class support for the CUDA language in CMake. Superseded by the FindCUDAToolkit for CUDA toolkit libraries.

> It is no longer necessary to use this module or call find_package(CUDA) for compiling CUDA code. Instead, list CUDA among the languages named in the top-level call to the project() command

>  https://cmake.org/cmake/help/v3.18/module/FindCUDA.html

Reply:
I find that it is needed even though it claims it is depreciated.

Reply:
Oh, it a pity! Do you know why? Do you think it is a bug? If so, maybe let them know about it by creating issue in https://gitlab.kitware.com/cmake/cmake/-/issues?